### ISSUE TITLE
RV fails to build with LLVM 5 non-debug builds

### ISSUE THREAD

**Issue by mppf** [cdl-saarland/rv#9]
I'm trying to build RV with LLVM 5 on Ubuntu 17.10, in a non-debug build. I get errors like this:

```
CMakeFiles/rvTool.dir/rvTool.cpp.o: In function `main':
rvTool.cpp:(.text.startup.main+0x9ed): undefined reference to `llvm::Module::dump() const'
../../../lib/libRV.a(LoopVectorizer.cpp.o): In function `rv::LoopVectorizer::vectorizeLoop(llvm::Loop&)':
LoopVectorizer.cpp:(.text._ZN2rv14LoopVectorizer13vectorizeLoopERN4llvm4LoopE+0x2508): undefined reference to `llvm::Value::dump() const'
LoopVectorizer.cpp:(.text._ZN2rv14LoopVectorizer13vectorizeLoopERN4llvm4LoopE+0x2659): undefined reference to `llvm::Value::dump() const'
collect2: error: ld returned 1 exit status
tools/rv/tools/CMakeFiles/rvTool.dir/build.make:134: recipe for target 'bin/rvTool' failed
make[6]: *** [bin/rvTool] Error 1
CMakeFiles/Makefile2:40841: recipe for target 'tools/rv/tools/CMakeFiles/rvTool.dir/all' failed
```

See also https://github.com/chapel-lang/chapel/pull/7496 for what we had to do with the Chapel front-end for this... Basically it amounts to replacing `myValue->dump()` with `myValue->print(dbgs(), true)`.

**Comment by simoll** [comment]
Thanks for the pointer! This is fixed by 5b10a53. I tested it with an LLVM 5.0 release build. I'll leave this open until you can confirm that the patch works for your exact configuration.

**Comment by mppf** [comment]
Working for me, thanks.
